src/fixedpoint.ts

```diff
@@ -11,17 +25,45 @@ export class FixedInt<T extends number> {
     this.decimals = decimals;
   }
 
+  /**
+   * Get the value of the fixed point number.
+   */
   get val() {
```

Comment:
nit: wdyt about calling `value` instead of `val`?

Reply:
originally had `value` but in use, it felt verbose. Personal opinion tho. Could go `value`!

Reply:
Easy either way. val makes me think of kotlin lol

Reply:
even more verbose, but i like a good `.getValue()` method